SwiftProtobuf.podspec

```diff
@@ -12,6 +12,7 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.13'
   s.tvos.deployment_target = '11.0'
   s.watchos.deployment_target = '4.0'
+  s.visionos.deployment_target = "1.0"
 
   s.cocoapods_version = '>= 1.7.0'
```

Comment:
Does this need to also get updated so `visionos` will be known by cocoapods?  i.e. - what version did they add visionOS support?

Reply:
ok yes, it was `1.13.0` https://github.com/CocoaPods/CocoaPods/issues/11961

Reply:
Hopefully that's not too new for folks.  Thanks!